lib/components/molecules/Table/Table.js

```diff
@@ -0,0 +1,56 @@
+// @flow
+/**
+ *
+ * Table
+ *
+ */
+import React from 'react';
+import type { Props } from './types';
+import styled from 'styled-components';
+
+import styles from './Table.style';
+
+type tableProps = {
+  className?: string,
+  data: Array,
+};
+
+const Table = ({ data, className, ...others }: tableProps): Node => {
+  const columnName = Object.keys(data[0]);
+
+  const headers = (
+    <tr>
+      {columnName.map((name, n) => (
+        // eslint-disable-next-line react/no-array-index-key
```

Comment:
This style of disabling errors is not appreciated, these checks have been configured for a purpose